diff --git app.js app.js
index 673e43b..172d8fa 100644
--- app.js
+++ app.js
@@ -57088,7 +57088,8 @@ App.serviceMetricsMapper = App.QuickDataMapper.create({
       STORM: [31],
       FALCON: [32],
       RANGER: [33],
-      SPARK: [34]
+      SPARK: [34],
+      EAGLE: [35]
     };
     if (quickLinks[item.ServiceInfo.service_name])
       finalJson.quick_links = quickLinks[item.ServiceInfo.service_name];
@@ -66814,6 +66815,13 @@ App.QuickLinks.FIXTURES = [
     site: 'spark-defaults',
     regex: '^(\\d+)$',
     default_http_port: 18080
+  },
+  {
+    id: 35,
+    label:'Eagle Web',
+    url:'%@://%@:9099/eagle-service',
+    service_id: 'EAGLE',
+    template:'%@://%@:9099/eagle-service'
   }
 ];
